BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -334,10 +334,44 @@
         ]]>
     </string>
 
+    <string name="search_help_text">
+        <![CDATA[
+        <h1><u>Hilfe zur Suche</u></h1>
+        <p>Sie befinden sich in der Suche. Hier können Sie nach Regalen, Büchern und Notizen suchen.
+        </p>
 
+        <h2>Sortieren</h2>
+        <p>Sie können ein Kriterium nach dem Sie ihre Suchergebnisse sortieren wollen auswählen.
```

Comment:
```suggestion
        <p>Sie können ein Kriterium, nach dem Sie ihre Suchergebnisse sortieren wollen, auswählen.
```

Reply:
Missing comma and specify here which criteria.

Reply:
The criterias are described below this sentence.